File: FizzBuzz.java
public class Main {
    public static void main(String[] args) {
        for (int n =1; n< 101; n++){
            if (CheckDivisibility(n, 15)) {
                System.out.println("FizzBuzz");
            } else if (CheckDivisibility(n, 3)) {
                System.out.println("Fizz");
            } else if (CheckDivisibility(n, 5)) {
                System.out.println("Buzz");
            } else{
                System.out.println(n);
            }
        }
    }
    static Boolean CheckDivisibility(int dividend, int divisor) {
        return dividend % divisor == 0;
    }
}

File: FizzBuzz.php
<?php
class FizzBuzz {
    public function printFizzBuzz()
    {
        for ($n = 1; $n <= 100; $n++) {
            if ($this->checkDivisibility($n, 15)) {
                echo("FizzBuzz");
            } elseif ($this->checkDivisibility($n, 3)) {
                echo("Fizz");
            } elseif ($this->checkDivisibility($n, 5)) {
                echo("Buzz");
            } else {
                echo $n;
            }
        }
    }
    public function checkDivisibility($dividend, $divisor)
    {
        return $dividend % $divisor == 0;
    }
}
$fizzBuzz = new FizzBuzz();
$fizzBuzz -> printFizzBuzz();

















